### ISSUE TITLE
[Term Entry] Java Strings: `.offsetByCodePoints()` Method

### ISSUE THREAD

**Issue by KTom101** [Codecademy/docs#1590]
### Reason/inspiration (optional)

Add a new term entry under the [Java: Strings](https://www.codecademy.com/resources/docs/java/strings) concept. This entry should include:

An explanation of the method.

A "Syntax" section with a ```pseudo code snippet showing the general syntax.

An "Example" section with a ```java code snippet showing how it works.

A markdown file named "offsetByCodePoints.md" should be added to a folder named "offsetByCodePoints" and placed in the terms folder located under content/java/concepts/strings/terms

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by Codewindows2023** [comment]
Hi
Can I work on this issue please?
Thanks
Lisa

**Comment by KTom101** [comment]
@KTom101 I assigned you, check out the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md) for more info on contributing.

Also, how did you come across the Codecademy Docs repo?

**Comment by SSwiniarski** [comment]
@Codewindows2023, I'm unassigning you because of inactivity so someone else can work on this.

**Comment by realmattimatt** [comment]
Hello, I'd like to try this. I'm working on the Git & Github lesson. 
Thank you. 
Matt

**Comment by SSwiniarski** [comment]
@realmattimatt You're assigned! You can check out the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md) for additional information on contributing.

See [Content Standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md)for an overview of styling tips that the [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) covers in detail.

After creating a PR, maintainer(s) (with the collaborator label) will add in comments/suggestions to address any revisions prior to approving.

**Comment by realmattimatt** [comment]
Got some information about offsetByCodePoints.
Where do I start my document? I use VS Code. Thank you.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by yangc95** [comment]
@realmattimatt Please refer to the issue description here:

> A markdown file named "offsetByCodePoints.md" should be added to a folder named "offsetByCodePoints" and placed in the terms folder located under content/java/concepts/strings/terms

Create the folder under terms and nest your markdown file there. Hope this helps!

**Comment by SSwiniarski** [comment]
@realmattimatt It's been two weeks. Are you still working on this PR. Do you need assistance?

**Comment by realmattimatt** [comment]
Yes I started a markdown file, got discouraged and worried I wasn’t gonna do it correctly. I added to it today and will commit what I have so far. My apologies, I am completely new to this. I would like to get this done and do more. Any and all feedback is welcome.
Thanks. Matt

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by realmattimatt** [comment]
I am trying to understand how  this works. I installed Java for VS Code. I have not taken a java course yet and I do not understand. Can you remove me from this and find something in python or javascript? I've taken classes on those. I deeply apologize, I think I seen java and misinterpreted as javascript. Thanks for your time. Matt.

**Comment by SSwiniarski** [comment]
Ok @realmattimatt, I'm taking you off of this. You can keep an eye on the issues until you see something you're comfortable with.

**Comment by SSwiniarski** [comment]
@white-hawk-73 You're assigned! You can check out the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md) for additional information on contributing.

See [Content Standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md)for an overview of styling tips that the [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) covers in detail.

After creating a PR, maintainer(s) (with the collaborator label) will add in comments/suggestions to address any revisions prior to approving.

**Comment by SSwiniarski** [comment]
@white-hawk-73 It's been two weeks. Are you still working on this issue? Do you need assistance?

**Comment by white-hawk-73** [comment]
@SSwiniarski Yes, I have started on this but got occupied elsewhere. If by Friday, you don't hear from me, you can unassign me from this issue. Thanks for your patience!

**Comment by SSwiniarski** [comment]
@white-hawk-73 should I unassign you now?

**Comment by white-hawk-73** [comment]
@SSwiniarski Yes, please. Thank you for your time. I am new to open source so I will look around and pick an issue that I am comfortable with.

**Comment by sumitdethe27** [comment]
Hey @SSwiniarski , is this issue available, I would like to work on it.

**Comment by SSwiniarski** [comment]
@sumitdethe27 You're assigned! You can check out the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md) for additional information on contributing.
See [Content Standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) for an overview of styling tips that the [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) covers in detail.
After creating a PR, maintainer(s) (with the collaborator label) will add in comments/suggestions to address any revisions prior to approving.

In addition, please head over to the [Docs Forum](https://discuss.codecademy.com/c/community/community-docs/2205) if you have any questions about Docs or reply to the thread on [Possible Content Discussion](https://discuss.codecademy.com/t/find-a-starting-point-possible-content-discussion/745868) to share ideas and collaborate with other contributors, maintainers, or super users to determine good future issues that will help expand Docs!

If this is your first time contributing, can you let us know how you came across this repo?
